README.md

```diff
@@ -123,7 +123,11 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<<<<<<< HEAD
+[![All Contributors](https://img.shields.io/badge/all_contributors-49-orange.svg?style=flat-square)](#contributors-)
+=======
 [![All Contributors](https://img.shields.io/badge/all_contributors-51-orange.svg?style=flat-square)](#contributors-)
+>>>>>>> c5a7ffadca3c33134d8f0f536e6b8329c292e4a4
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
We don't want to modify this, but there's badge duplication here.
So, delete line 126-128 and 130.